packages/jest/src/__tests__/matchers.test.tsx

```diff
@@ -158,7 +164,7 @@ describe('toHaveCompliedCss', () => {
       </div>
     );
     const el = getByText('hello world');
-    expect(el).toHaveCompiledCss('background', 'red', { media: 'screen', target: ':hover' });
+    expect(el).toHaveCompiledCss('background-color', 'red', { media: 'screen', target: ':hover' });
```

Comment:
now the matcher will fail if you assert a shorthand - potentially. what should we do about this? (in another PR of course)

two thoughts are:

1. nothing - leave it as is and document the gotcha
2. convert the matcher to use post css and run it through the expand plugin 

🤔

Reply:
How about disabling it by default and let the user configure it in babel-plugin i.e. make it configurable 🤔? 
```typescript
expandShorthands: true/false
```

Probably we can also make other postcss plugins configurable (which are configurable)

Reply:
we need to be careful about this kind of configuration because we want to ensure a consistent experience when it comes to interop between usages (e.g. atlassian frontend vs. jira)

for now let's not?

Reply:
Yeah just a suggestion 😃 I mean in case people really don't care about shorthands..if we are using compiled outside of Atlassian.
Like below where order doesn't matter. Both will work. But this plugin will reorder it.
```typescript
expect(actual).toInclude(`{border:1px solid black}`);  => expect(actual).toInclude(`{border:1px black solid}`);
```

Reply:
I am in favour of "convert the matcher to use post css and run it through the expand plugin". Because we are giving our own matcher `toHaveCompiledCss`. What you say 🤔?

Reply:
i'll make a issue :)

Reply:
raised https://github.com/atlassian-labs/compiled/issues/321